packages/react-google-maps-api/src/components/dom/OverlayView.tsx

```diff
@@ -140,6 +140,12 @@ export class OverlayView extends React.PureComponent<
 
     this.setState(setOverlayView, this.setOverlayViewCallback)
   }
+  
+  componentDidUpdate(prevProps: OverlayViewProps) {
```

Comment:
@FredyC I guess we need to add ```getPixelPositionOffset``` to this check no? 
Also, why the setTimeout? I see the original lib does it too but not sure why...

Reply:
Yea, I did it based on original considering it was working. Hard to say what is the reasoning behind it.

Reply:
I can only guess he didn't want it to block rendering..  Though, we're not really rendering anything in the component besides it's children. I'll need to test it out but it feels like this component could have

```
shouldComponentUpdate() {
   return false;
}
```

Or maybe just re-render only if the children prop changed

Reply:
I don't think you can compare children prop, it's always going to be a different object unless someone would actually `React.useMemo` it :) Also `getPixelPositionOffset` might in theory be dynamic. I wouldn't over optimize here, the `PureComponent` should suffice or later `React.memo` if we get rid of the class.